### DISCUSSION TITLE
希望能在 搜索页 无缝加载 并 进一步排序/筛选

### DISCUSSION THREAD

**Discussion by wlm3201** [the1812/Bilibili-Evolved#4583] [功能建议 / Ideas]
在搜索页综合排序下随滚动条无缝加载后续页面并对已加载的视频进一步按显示的元素如播放量/评论数/时长/发布时间排序/筛选

**Comment by the1812** [comment]
这种需求请去找无限翻页脚本, 比如 https://greasyfork.org/zh-CN/scripts/438684-pagetual
